src/univariate/continuous/maxwellboltzmann.jl

```diff
@@ -0,0 +1,63 @@
+"""
+    MaxwellBoltzmann(a)
+
+The *Maxwell-Boltzmann distribution* has probability density function
+```math
+f(x; a) = \\sqrt{\\frac{2}{\\pi}}\\frac{x^2}{a^3}\\exp\\left(-\\frac{x^2}{2a^2}\\right)
+```
+where the parameter
+```math
+a = \\sqrt{\\frac{kT}{m}}
+```
+
+```julia
+MaxwellBoltamann()        # Maxwell-Boltzmann distribution with a = 1
+MaxwellBoltzmann(a)       # Maxwell-Boltzmann distribution with chosen parameter a
+MaxwellBoltzmann(T, m)    # Maxwell-Boltzmann distribution with chosen temperature T and mass m
+```
+
+External links
+
+* [Maxwell-Boltzmann distribution on Wikipedia](https://en.wikipedia.org/wiki/Maxwell-Boltzmann_distribution)
+"""
+struct MaxwellBoltzmann{T<:Real} <: ContinuousUnivariateDistribution
+    a::T
+    MaxwellBoltzmann{T}(a::T) where {T} = new{T}(a)
+end
+
+function MaxwellBoltzmann(a::T; check_args=true) where {T<:Real}
+    check_args && @check_args(MaxwellBoltzmann, a > zero(a))
+    return MaxwellBoltzmann{T}(a)
+end
+
+MaxwellBoltmann() = MaxwellBoltzmann(1.0)
```

Comment:
```suggestion
MaxwellBoltzmann() = MaxwellBoltzmann(1.0)
```
There's a typo here.